<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>asyncio on The Knife of Phaedrus</title>
    <link>https://lafolle.ca/tags/asyncio/</link>
    <description>Recent content in asyncio on The Knife of Phaedrus</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <copyright>All content copyright lafolle.ca © 2020 • All rights reserved.</copyright>
    <lastBuildDate>Thu, 10 Mar 2022 22:08:29 -0800</lastBuildDate><atom:link href="https://lafolle.ca/tags/asyncio/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>On asyncio run</title>
      <link>https://lafolle.ca/blog/asyncio_loop/</link>
      <pubDate>Thu, 10 Mar 2022 22:08:29 -0800</pubDate>
      
      <guid>https://lafolle.ca/blog/asyncio_loop/</guid>
      <description>Python formally introduced concurrency using asncio in 3.7+. A simple program that can demonstrate the hello world of async stuff goes like this-
import asyncio async def main(): print(&amp;#39;hello&amp;#39;) await asyncio.sleep(1) print(&amp;#39;world&amp;#39;) asyncio.run(main()) In this case asyncio.run is the blocking call. It takes coroutine as a param, that is created by main() due to the async qualifer.
Does it also create an event loop inside? From the doc, yes it does-</description>
    </item>
    
  </channel>
</rss>
